packages/mobile/src/hooks/useIsUSDCEnabled.ts

```diff
@@ -0,0 +1,6 @@
+import { FeatureFlags } from '@audius/common'
+
+import { useFeatureFlag } from './useRemoteConfig'
+
+export const useIsUSDCEnabled = () =>
```

Comment:
nit: personally don't see the need of this hook wrapper, calling `useFlag`/`useFeatureFlag` seems less indirect

Reply:
personally like this abstraction - fewer imports, no need to think about the feature flag API or enums when you just want to enable/disable some code

Reply:
I agree. I was following our feature flag guide, which requests that if you're going to use a flag in more than a couple of places (which we definitely will), that you create a function for reading it.

But maybe that was written before the hooks existed?

Reply:
Guess I'm the odd one out 😅 it just seems like a lot of code to turn a one liner into another one liner